hooks/post_push

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
could you add a new README.md to the hooks directory that explains that it's used for dockerhub?